test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart_default.rhel9_dhcp.snap.txt

```diff
@@ -0,0 +1,197 @@
+# This kickstart file was rendered from the Foreman provisioning template "Kickstart default".
+# for snapshot-ipv4-dhcp-rhel9 running RHEL 9 x86_64
+# Organization: Organization 1
+# Location: Location 1
+
+
+url --url http://mirror.example.com/rhel/9/os/x86_64
+
+
+lang en_US.UTF-8
+selinux --enforcing
+keyboard us
+
+network --device=00-f0-54-1a-7e-e0 --hostname snapshot-ipv4-dhcp-rhel9 --noipv6 --bootproto dhcp --mtu=1142 --nameserver=192.168.42.2,192.168.42.3
+
+rootpw --iscrypted $1$rtd8Ub7R$5Ohzuy8WXlkaK9cA2T1wb0
+firewall --service=ssh
+authselect --useshadow --passalgo=sha256 --kickstart
+timezone --utc UTC 
+
+services --disabled gpm,sendmail,cups,pcmcia,isdn,rawdevices,hpoj,bluetooth,openibd,avahi-daemon,avahi-dnsconfd,hidd,hplip,pcscd
+
+
+
+bootloader --location=mbr --append="nofb quiet splash=quiet" 
+
+zerombr
+clearpart --all    --initlabel
+part /boot --fstype ext3 --size=100 --asprimary
+part /     --fstype ext3 --size=1024 --grow
+part swap  --recommended
+rhsm --organization="Org" --activation-key="key" --connect-to-insights
+syspurpose --role 'Red Hat Enterprise Linux Server' --usage 'Development/Test' --sla 'Self-Support' --addon 'first addon' --addon 'second addon' --addon 'third addon'
+
+skipx
+text
+reboot
+
+%packages
+
+yum
+dhclient
+chrony
+-ntp
+@Core
+dracut-fips
+-prelink
+
+%end
+
+
+%post --nochroot
+exec < /dev/tty3 > /dev/tty3
+chvt 3
+(
+
+
+
+chvt 1
+) 2>&1 | tee /mnt/sysimage/root/install.postnochroot.log
+%end
+
+
+%post
+exec < /dev/tty3 > /dev/tty3
+chvt 3
+(
+logger "Starting anaconda snapshot-ipv4-dhcp-rhel9 postinstall"
+
+
+
+
+
+echo "Updating system time"
+systemctl enable --now chronyd
+/usr/bin/chronyc -a makestep
+/usr/sbin/hwclock --systohc
+
+
+
+
+
+rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+
+
+
+# update all the base packages from the updates repository
+if [ -f /usr/bin/dnf ]; then
+  dnf -y update
+else
+  yum -t -y update
+fi
```

Comment:
I wonder if this is still needed when registration happens. I'd imagine it pulls all the latest versions while installing. Correct @M4rtinK?